### PR TITLE
Backport all doc PRs up to 0.13.0 release

### PR DESCRIPTION
**Pull request by jackye1995**
as a part of https://github.com/apache/iceberg/pull/4056

@samredai @rdblue

### CODE REVIEW

File: docs/content/docs/spark/spark-procedures.md [hunk 1]
```diff
@@ -340,6 +354,15 @@ CALL catalog_name.system.rewrite_manifests('db.sample', false)
 
 The `snapshot` and `migrate` procedures help test and migrate existing Hive or Spark tables to Iceberg.
 
+{{< hint info >}}
```
Comment (rdblue) [inline docs/content/docs/spark/spark-procedures.md:357]: We should be able to remove this warning now that the fix is released, right?

**Review by rdblue — APPROVED** [review]

**Comment by rdblue** [comment]
Looks great! I think we can remove an outdated hint, but this is ready to go. Merge when you're ready!

**Comment by samredai** [comment]
lgtm, thanks Jack!

File: docs/content/docs/spark/spark-procedures.md (hunk 1 above)
Comment (jackye1995) [reply docs/content/docs/spark/spark-procedures.md:357]: yes you are right, updated

**Comment by jackye1995** [comment]
Thanks for the review! @rdblue @samredai

**Pull request merged**